repository/templates/base.html

```diff
@@ -120,11 +120,9 @@ <h2 class="mysoc-footer__site-name"><a href="https://mysociety.org?utm_source=wh
             <div class="col-sm-9">
                 <div class="mysoc-footer__legal">
                     <p>
-                        <a href="https://www.mysociety.org?utm_source=research.mysociety.org&amp;utm_content=footer+full+legal+details&amp;utm_medium=link&amp;utm_campaign=mysoc_footer">mySociety</a>
-                        is a registered charity in England and Wales (1076346)
-                        and a limited company (03277032). Through our trading
-                        subsidiary SocietyWorks (05798215) we also provide
-                        commercial services.
+                        <a href="https://www.mysociety.org?utm_source=research.mysociety.org&amp;utm_content=footer+full+legal+details&amp;utm_medium=link&amp;utm_campaign=mysoc_footer"></a> is a registered charity in England and Wales (1076346) 
```

Comment:
No text in the link.

Reply:
Sorry. I've corrected it in the branch now - is that all I need to do?